server.js

```diff
@@ -6,6 +6,7 @@ const winston = require("winston");
 const { combine, timestamp, json, errors } = winston.format;
 const morgan = require("morgan");
 const rateLimit = require("express-rate-limit");
+// const { LoggingWinston } = require("@google-cloud/logging-winston");
```

Comment:
Are those comments expected?

Reply:
yeah I left them in because they'll just be uncommented in a follow-up once Quentin sets up the SA for the new gcp project.